zsh-aws-vault.plugin.zsh

```diff
@@ -18,12 +19,34 @@ alias avll='avl -s'
 # Convenience Functions                                              #
 #--------------------------------------------------------------------#
 function avsh() {
-  aws-vault exec $1 -- zsh
+  case ${AWS_VAULT_PL_MFA} in
+    inline)
+      aws-vault exec -t $2 $1 -- zsh
+      ;;
+    yubikey)
+      totp=${2:$1}
```

Comment:
I think you've got a typo here. My assumption is that you want to default to the profile name (`$1`).

```suggestion
      totp=${2:-$1}
```